### PR TITLE
Add guest_search function for nutanix automation support

### PR DESCRIPTION
**Pull request by hkx303**
Add guest_search function for nutanix automation support

### CODE REVIEW

File: hypervisor/virt/ahv/ahvncli.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        self.server = server
+        self.ssh_user = ssh_user
+        self.ssh_pwd = ssh_pwd
+        self.ssh = SSHConnect(self.server, user=self.ssh_user, pwd=self.ssh_pwd)
+
+    def get_ncli_info(self, cmd, value):
+        """
+        Get the info from AHV
+        :param cmd:
+        :param value: the value you want to get from the result
```
Comment (Junefen) [inline hypervisor/virt/ahv/ahvncli.py]: I think we should give an example for the both cmd and value,

File: hypervisor/virt/ahv/ahvncli.py [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+            return None
+
+    def guest_search(self, guest_name):
+        """
+        Search the specific guest, return the expected attributes
+        :param guest_name: name for the specific guest
+        :param host_ip: the ip for the AHV host
+        :param host_user: the ssh username for the AHV host
+        :param host_pwd: the ssh password for the AHV host
+        :return: guest attributes, exclude guest_name, guest_ip, guest_uuid ...
```
Comment (Junefen) [inline hypervisor/virt/ahv/ahvncli.py]: exclude?

File: hypervisor/virt/ahv/ahvncli.py [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+            logger.info(f"Failed to get {value}")
+            return None
+
+    def guest_search(self, guest_name):
+        """
+        Search the specific guest, return the expected attributes
+        :param guest_name: name for the specific guest
+        :param host_ip: the ip for the AHV host
+        :param host_user: the ssh username for the AHV host
+        :param host_pwd: the ssh password for the AHV host
```
Comment (Junefen) [inline hypervisor/virt/ahv/ahvncli.py]: It seems this function only uses one parameter ```guest_name```

File: hypervisor/virt/ahv/ahvncli.py (hunk 1 above)
Comment (hkx303) [reply hypervisor/virt/ahv/ahvncli.py]: will update this function with another suitable way :)

File: hypervisor/virt/ahv/ahvncli.py (hunk 2 above)
Comment (hkx303) [reply hypervisor/virt/ahv/ahvncli.py]: my bad, include actually 0-0

File: hypervisor/virt/ahv/ahvncli.py (hunk 3 above)
Comment (hkx303) [reply hypervisor/virt/ahv/ahvncli.py]: will update the docstring later

**Comment by hkx303** [comment]
Hey @Junefen, plz add the `[ahv]` part in https://github.com/VirtwhoQE/virtwho-test/blob/master/virtwho.ini.sample, so I will add the params in the `guest_search` function, thanks!

**Pull request merged**